tests/cpp-tests/Classes/UITest/CocoStudioGUITest/UIWebViewTest/UIWebViewTest.cpp

```diff
@@ -144,6 +148,40 @@ bool WebViewTest::init()
         evalJsBtn->setName("evalJs");
         this->addChild(evalJsBtn);
         
+        Button *opacityBtn = Button::create("cocosui/animationbuttonnormal.png",
+                                           "cocosui/animationbuttonpressed.png");
+        opacityBtn->setTitleText("Opacity 1.f");
+        opacityBtn->setPosition(Vec2(winSize/2) - Vec2( _webView->getContentSize().width/2 +
+                                                      opacityBtn->getContentSize().width/2 + 10, 100 ));
+        opacityBtn->addClickEventListener([=](Ref*){
+            auto currentOpacity = _webView->getOpacityWebView();
+            if(currentOpacity ==1.f){
+                _webView->setOpacityWebView(.5f);
+                opacityBtn->setTitleText("Opacity .5f");
+            }else if(currentOpacity == .5f){
+                _webView->setOpacityWebView(0);
+                opacityBtn->setTitleText("Opacity 0.f");
+            }else{
+                _webView->setOpacityWebView(1.f);
+                opacityBtn->setTitleText("Opacity 1.f");
+            }
+            
+        });
+        opacityBtn->setName("Opacity");
+        this->addChild(opacityBtn);
+        
+        
+        Button *transparentBgBtn = Button::create("cocosui/animationbuttonnormal.png",
+                                            "cocosui/animationbuttonpressed.png");
+        transparentBgBtn->setTitleText("Transparent BG");
+        transparentBgBtn->setPosition(Vec2(winSize/2) + Vec2( _webView->getContentSize().width/2 +
+                                                             transparentBgBtn->getContentSize().width/2 + 10,-100 ));
+        transparentBgBtn->addClickEventListener([=](Ref*){
+            _webView->setBackgroundTransparent();
```

Comment:
It seems there is not different on Android, or may be the test case can not show it?

Reply:
I think that it depends of the page. I found websites that doesnt make the transparency also. I dont really know. 
`webView.setBackgroundColor(Color.TRANSPARENT);
webView.setLayerType(WebView.LAYER_TYPE_SOFTWARE, null);`
These are the used methods from Android.

Reply:
Got it, thanks. Have you used it in your game?